examples/run_CPMG.py

```diff
@@ -10,116 +9,128 @@
 from pyspecdata import *
 from numpy import *
 import SpinCore_pp
-from SpinCore_pp.ppg import run_cpmg
+from SpinCore_pp import prog_plen
+from SpinCore_pp.ppg import generic
 import os
 from datetime import datetime
 import h5py
-raise RuntimeError("This pulse proram has not been updated.  Before running again, it should be possible to replace a lot of the code below with a call to the function provided by the 'generic' pulse program inside the ppg directory!")
+from Instruments.XEPR_eth import xepr
 
-fl = figlist_var()
 # {{{importing acquisition parameters
 config_dict = SpinCore_pp.configuration("active.ini")
-nPoints = int(config_dict["acq_time_ms"] * config_dict["SW_kHz"] + 0.5)
+nPoints = int(config_dict["echo_acq_ms"] * config_dict["SW_kHz"] + 0.5)
```

Comment:
in generic.py, this is the points *per echo* so you should have a comment here clarifying that.  Also, later down when you  calculate the total number of points, you *do* want to multiply by the number of echoes.